design/src/high-level/game-thesis.md

```diff
@@ -0,0 +1,176 @@
+  - flying units can travel more freely, but have lower carrying loads and much higher upkeep
+  - water flows can be used to carry goods downriver
+  - large quantities of goods can be transported via large, high-momentum ground units
+- **Storage**
+  - simple one-resource piles that are exposed to the elements
+  - sheltered storage buildings made of plants and fungi
+  - units and storage work together in a fashion directly analogous to Factorio's logistic network
+- **Resource sinks**
+  - lossy conversions: energy and water will commonly be lost during resource transformations
+  - resource upkeep: energy and water will commonly be lost
+  - pheromones: used to exert control over the colony and interactively boost effectiveness
+  - guided evolution: research analogue used to modify and enhance existing species
+  - assimilation: research analogue used to add new species to the colony
+  
+### Advanced Mechanics
+
+- **Distributed resource costs**
+  - water is used by everything, with rate varying by temperature and humidity
+    - ultimately replenished by rainfall
+    - distributed via:
+      - canals
+      - mycorrhizal networks
+      - water droplets
+  - energy is used by everything, with rate varying by amount of work done
+    - ultimately gathered via photosynthesis
+    - distributed by:
+      - items
+      - mycorrhizal networks
+- **Fluids**
+  - transported via plant and fungal networks
+  - transported via canals
+  - can be pumped
+- **Filters**
+  - units are capable of differentiation
+  - simple mechanical filters sort items into classes
+    - item size
+    - floats
+    - blows away
+  - plants are capable of sophisticated filtration of solutions
+- **Bypasses**
+  - underground tunnels
+  - overpasses
+  - catapults?
+  - bridges?
+- **Spatial constraints**
+  - rocks
+  - trees
+  - bodies of water
+  - litter from extinct humans?
+  - modifying the topology etc should be possible, but very expensive
+- **Technology**
+  - guided evolution: research analogue used to modify and enhance existing species
+  - assimilation: research analogue used to add new species to the colony
+- **Production enhancements**
+  - pheromones: spent manually by the player to temporarily and locally enhance
+  - upgrades: done via guided evolution, affecting all organisms of that strain
+  - enhancements: pheromones can be produced and applied automatically via workers
+
+### Supplementary Mechanics
+
+These features supplement the core gameplay loop by providing additional things to do or consider, but are not needed.
+
+- **Exploration**
+  - important but not essential
+  - fairly standard implementation
+  - needs more interesting world generation
+  - water distribution and topography are key
+- **Combat**
+  - natural but not essential
+  - must be careful to avoid snowballing effects
+    - design with negative feedback loops like hunger satiation
+- **Pollution**
+  - thematically essential
```

Comment:
Pollution: changes the environmental assumptions that the hive depends on to keep on functioning? 

If so, it can be imagined as yet another dynamic constraint (similar to weather, etc.)

Reply:
Oh that's a nice framing.